### PR TITLE
Update solo/atmos_model.F90 for FMS module new aliasing

### PR DESCRIPTION
**Pull request by laurenchilutti**
**Description**
solo/atmos_model.F90 uses the FMS module.  In FMS 2023.02-beta1 this module has aliased all functions to contain a prefix before the function name.  This PR changes to the new naming convention so that the code can compile with FMS 2023.02-beta2.

Fixes # (issue)

**How Has This Been Tested?**
Tested with SHiELD solo tests on Gaea with intel

**Checklist:**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules
- [x] New check tests, if applicable, are included

### CODE REVIEW

**Review by mcallic2 — APPROVED** [review]

File: solo/atmos_model.F90 [hunk 1]
```diff
@@ -124,59 +124,59 @@ subroutine atmos_model_init
...
 !  (will print number below)
-   call register_tracers ( MODEL_ATMOS, ntrace, ntprog, ntdiag, ntfamily )
+   call fms_tracer_manager_register_tracers ( MODEL_ATMOS, ntrace, ntprog, ntdiag, ntfamily )
 
 
 !----- read namelist -------
 
-   read (input_nml_file, nml=main_nml, iostat=io)
+   read (fms_mpp_input_nml_file, nml=main_nml, iostat=io)
    ierr = check_nml_error(io, 'main_nml')
```
Comment (bensonr) [inline solo/atmos_model.F90:150]: Did we change check_nml_error as well?

File: solo/atmos_model.F90 [hunk 2]
```diff
@@ -124,59 +124,59 @@ subroutine atmos_model_init
...
 
 !----- write namelist to logfile -----
 
-   call write_version_number ("program atmos_model", version)
-   if ( mpp_pe() == mpp_root_pe() ) write (logunit, nml=main_nml)
+   call fms_write_version_number ("program atmos_model", version)
+   if ( fms_mpp_pe() == fms_mpp_root_pe() ) write (logunit, nml=main_nml)
 
    if (dt_atmos == 0) then
      call error_mesg ('program atmos_model', 'dt_atmos has not been specified', FATAL)
```
Comment (bensonr) [inline solo/atmos_model.F90:158]: Did we change error_mesg as well?

File: solo/atmos_model.F90 (hunk 1 above)
Comment (laurenchilutti) [reply solo/atmos_model.F90:150]: No, this was not aliased in libFMS.F90

File: solo/atmos_model.F90 (hunk 2 above)
Comment (laurenchilutti) [reply solo/atmos_model.F90:158]: No, this also wasnt changed.

**Review by bensonr — APPROVED** [review]

**Comment by laurenchilutti** [comment]
I will merge this now and create a 2023.02-beta1 tag for testing.  I have created an issue to bring attention to the tow routines that were not aliased. And if there are more changes in FMS, then I will create another PR to address that.

**Pull request merged**